src/control.rs

```diff
@@ -43,14 +43,45 @@ impl ShouldColorize {
     pub fn from_env() -> Self {
         use std::io;
 
+        // Treat NO_COLOR as an alias for CLICOLOR_FORCE == 0
+        let clicolor_force = match (
+            ShouldColorize::normalize_env(env::var("NO_COLOR")),
+            ShouldColorize::normalize_env(env::var("CLICOLOR_FORCE")),
+        ) {
+            // Give priority to an explicitly set CLICOLOR_FORCE.
+            (_, Some(b)) => Some(b),
+
+            // Otherwise, NO_COLOR being set should act as CLICOLOR_FORCE == 0
+            (Some(_), None) => Some(false),
+
+            (None, None) => None,
+        };
```

Comment:
could you:
1. extract this block to a private function
2. test it with the few unit tests to cover the match logic
3. remove the comments (documentation being now in the executable tests, so it's easier for the maintainance)